contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -48,7 +48,7 @@ new Vue ({
 });
 ```
 
-Step-by-step explanation of the code:
+++Step-by-step explanation of the code++
```

Comment:
Best not to use underlines as it makes text harder to read, and also, can be confused as a link.